docs/config/README.md

```diff
@@ -1179,6 +1180,37 @@ The module will be shown if any of the following conditions are met:
 symbol = "🐪 "
 ```
 
+## Perl
+
+The `perl` module shows the currently installed version of Perl.
+The module will be shown if any of the following conditions are met:
+
+- The current directory contains a `Makefile.PL` file
+- The current directory contains a `cpanfile` file
```

Comment:
Well, if you support `cpanfile` then you should probably support  `cpanfile.snapshot` too. That's pretty much CPAN's version of a lockfile.

Reply:
added, doc updated, though i think there has to be a cpanfile if there is a cpanfile.snapshot, not sure, it wont hurt anyhow.

Reply:
> added, doc updated, though i think there has to be a cpanfile if there is a cpanfile.snapshot, not sure, it wont hurt anyhow.

That would be pretty unusual, I agree... But as you said, won't hurt 🙃